src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -3186,10 +3184,10 @@ void GraphOptimizer::RemoveConvertMemoryOutput(Graph& graph) {
         }
 
         auto&& childEdges = node->getChildEdgesAtPort(0);
-        for (auto&& edge : childEdges) {
-            if (Type::MemoryOutput != edge->getChild()->getType()) {
-                return false;
-            }
+        if (!std::all_of(childEdges.begin(), childEdges.end(), [](const auto& edge) {
```

Comment:
Let's have a named lambda for better readability.

Reply:
Done